.Lib9c.Tests/Model/State/WeeklyArenaStateTest.cs

```diff
@@ -205,7 +226,18 @@ public void GetArenaInfosByUpperAndLowerRange(int infoCount, int targetRank, int
             }
 
             var arenaInfos = weeklyArenaState.GetArenaInfos(targetAddress, upperRange, lowerRange);
-            Assert.Equal(expected, arenaInfos.Count);
+            Assert.Equal(expectedCount, arenaInfos.Count);
+
+            if (!arenaInfos.Any())
```

Comment:
is this correct changes?

Reply:
Looking back, it's an unnecessary conditional statement.